service/javadsl/server/src/main/java/com/lightbend/lagom/javadsl/server/ServiceGuiceSupport.java

```diff
@@ -32,15 +52,23 @@ default void bindServices(ServiceBinding<?>... serviceBindings) {
             }
         }
 
+        ServiceBinding<?> primaryServiceBinding = serviceBindings[0];
         // Bind the service info for the first one passed in
-        binder.bind(ServiceInfo.class).toProvider(new ServiceInfoProvider(serviceBindings[0].serviceInterface()));
+        binder.bind(ServiceInfo.class).toProvider(
+                new ServiceInfoProvider(
+                        primaryServiceBinding.serviceInterface(),
+                        Arrays
+                                .stream(serviceBindings)
+                                .map(ServiceBinding::serviceInterface)
+                                .toArray(Class[]::new)
+                ));
```

Comment:
Internal services don't make it to ServiceInof in any case. We just know they are there but, IIUC, at the moment, we don't make a lot of fuss about them...